src/forms/CurrentIncome.js

```diff
@@ -130,11 +144,20 @@ const IncomeForm = function ({ current, time, setClientProperty }) {
 * @returns Component
 */
 // `props` is a cloned version of the original props. References broken.
+/** @todo  Make currentIncomeStep compatible with CurrentBenefitsStep? 
+ *         currentBenefits looks like this:
+ *         const CurrentBenefitsStep = ({ changeClient, nextStep, client, snippets })
+ * Which one is correct?
+*/
 const CurrentIncomeStep = function (props) {
 
   const setTimeProp = getTimeSetter('current', props.changeClient);
+  const snippets = props.snippets;
 
-  /** @todo Are these titles accurate now? */
+  /** @todo Are these titles accurate now? 
+   * @todo Change phrase after title to snippets.title after changing en.js?
+   * @todo Change phrase after clarifier to snippets.clarifier after changing en.js?
+  */
```

Comment:
Why don't you go ahead and add those into `en.js`, so that it all gets done in one pass?